2021/01/14 17:18:10 : Error Connecting:HTTPConnectionPool(host='127.0.1.1', port=8080): Max retries exceeded with url: /api/v1/action/setup_cart (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7fb6bd946940>: Failed to establish a new connection: [Errno 111] Connection refused',)) : False
2021/01/14 17:19:33 : Error Connecting:HTTPConnectionPool(host='127.0.1.1', port=8080): Max retries exceeded with url: /api/v1/action/setup_cart (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7fd8826f64a8>: Failed to establish a new connection: [Errno 111] Connection refused',)) : False
2021/01/14 17:20:39 : Error Connecting:HTTPConnectionPool(host='127.0.1.1', port=8080): Max retries exceeded with url: /api/v1/action/setup_cart (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f3338cd54e0>: Failed to establish a new connection: [Errno 111] Connection refused',)) : False
2021/01/16 13:43:39 : Error Connecting:HTTPConnectionPool(host='127.0.1.1', port=8080): Max retries exceeded with url: /api/v1/action/setup_cart (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7facf91b85f8>: Failed to establish a new connection: [Errno 111] Connection refused',)) : False
2021/01/16 15:12:54 : Error Connecting:HTTPConnectionPool(host='127.0.1.1', port=8080): Max retries exceeded with url: /api/v1/action/setup_cart (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f93ff8e55f8>: Failed to establish a new connection: [Errno 111] Connection refused',)) : False
2021/01/18 13:59:55 : Error Connecting:HTTPConnectionPool(host='127.0.1.1', port=8080): Max retries exceeded with url: /api/v1/action/setup_cart (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7ff41129d3c8>: Failed to establish a new connection: [Errno 111] Connection refused',)) : False
2021/01/18 14:11:21 : Error Connecting:HTTPConnectionPool(host='127.0.1.1', port=8080): Max retries exceeded with url: /api/v1/action/setup_cart (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f0eabf4c5f8>: Failed to establish a new connection: [Errno 111] Connection refused',)) : False
2021/01/18 16:43:28 : Error Connecting:HTTPConnectionPool(host='127.0.1.1', port=8080): Max retries exceeded with url: /api/v1/action/setup_cart (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f68e37735f8>: Failed to establish a new connection: [Errno 111] Connection refused',)) : False
2021/01/19 08:41:54 : Error Connecting:HTTPConnectionPool(host='127.0.1.1', port=8080): Max retries exceeded with url: /api/v1/action/setup_cart (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f920d8354a8>: Failed to establish a new connection: [Errno 111] Connection refused',)) : False
2021/01/19 11:07:25 : Error Connecting:HTTPConnectionPool(host='127.0.1.1', port=8080): Max retries exceeded with url: /api/v1/action/setup_cart (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7fb4520bc5f8>: Failed to establish a new connection: [Errno 111] Connection refused',)) : False
2021/01/20 10:02:33 : Error Connecting:HTTPConnectionPool(host='127.0.1.1', port=8080): Max retries exceeded with url: /api/v1/action/setup_cart (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7fcd1fc93630>: Failed to establish a new connection: [Errno 111] Connection refused',)) : False
2021/01/20 15:17:24 : Error Connecting:HTTPConnectionPool(host='127.0.1.1', port=8080): Max retries exceeded with url: /api/v1/action/setup_cart (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f43aea823c8>: Failed to establish a new connection: [Errno 111] Connection refused',)) : False
2021/01/21 09:17:02 : Error Connecting:HTTPConnectionPool(host='127.0.1.1', port=8080): Max retries exceeded with url: /api/v1/action/setup_cart (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f8d602066d8>: Failed to establish a new connection: [Errno 111] Connection refused',)) : False
2021/01/21 09:17:25 : Error Connecting:HTTPConnectionPool(host='127.0.1.1', port=8080): Max retries exceeded with url: /api/v1/action/setup_cart (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f0cd371c5f8>: Failed to establish a new connection: [Errno 111] Connection refused',)) : False
2021/01/21 09:17:33 : Error Connecting:HTTPConnectionPool(host='127.0.1.1', port=8080): Max retries exceeded with url: /api/v1/action/setup_cart (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f56679db5f8>: Failed to establish a new connection: [Errno 111] Connection refused',)) : False
